### PR TITLE
Update progress_bar for PL 1.5.5 & change 'target' abbreviation

### PR DESCRIPTION
**Pull request by XianyuanLiu**
### Description
1. Update `progress bar` in `action_dann` and `digit_dann`. The old way will be expired in PyTorch lightning 1.7.
2. Change parameter type from 'int' to 'float' in `(video_)domain_adapter` to avoid type mismatching.
3. Change 'target' abbreviation from 'tar' to 'tgt'. 
4. Change `ReverseLayerF` to `GradReverse`.
5. Simplify function/class names in `(video_)domain_adapter`.
6. Unify validation abbreviation to "val".

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] In-line docstrings updated.
- [ ] [Source for documentation at `docs`](https://github.com/pykale/pykale/tree/main/docs/source) manually updated for new API.

### CODE REVIEW

File: examples/digits_dann_lightn/config.py [hunk 1]
```diff
@@ -46,7 +46,7 @@
 # Adaptation-specific solver config
 _C.SOLVER.AD_LAMBDA = True
 _C.SOLVER.AD_LR = True
-_C.SOLVER.INIT_LAMBDA = 1
+_C.SOLVER.INIT_LAMBDA = 1.0
```
Comment (sz144) [inline examples/digits_dann_lightn/config.py:49]: Shall we search and make changes for all examples?

**Comment by sz144** [comment]
For changes 1 and 2, shall we search and make changes for all examples?

**Comment by XianyuanLiu** [comment]
> For changes 1 and 2, shall we search and make changes for all examples?

I think so. I have changed `action-da`, `digit-da` and `multisource-da` examples. I don't change other examples and the notebook. I would like to ask Shuo for help with the notebook.

Change 2 still has a warning about the type of `last epoch`. `last epoch` is generated from PL, so I don't change it. I only change the parameters we set. Change 3 & 5 are for avoiding misunderstanding. Change 4 is for unifying.

**Review by haipinglu — APPROVED** [review]
Well done Xianyuan! Much better quality now. This is your strength!
Sorry for the delay in review.

**Pull request merged**